### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by NSeydoux**
This defines the code owners for the repository. For more information, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @inrupt/sdk-owners @inrupt/apps-owners
```
Comment (matthieubosquet) [inline CODEOWNERS:1]: Should app-owners be here too?

**Review by matthieubosquet — APPROVED** [review]
LGTM

**Review by ajacksified — CHANGES_REQUESTED** [review]
Why `@inrupt/apps-owners?`

File: CODEOWNERS (hunk 1 above)
Comment (NSeydoux) [reply CODEOWNERS]: Note, that's a type, fixed in 17dbb67b2d152e08c906805deed3b177777ba356

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -29,7 +29,9 @@ jobs:
     - run: npm run build
     - run: npm run lint
     - run: npm run check-licenses
-    - run: npm audit --audit-level=moderate
+    # Set back to --audit-level=moderate when https://github.com/advisories/GHSA-qrpm-p2h7-hrv2
+    # is resolved in testcafe
```
Comment (matthieubosquet) [inline .github/workflows/ci.yml:33]: Or we’ve moved to playright ^^,

**Review by ajacksified — APPROVED** [review]

**Pull request merged**